xml/ses_glossary.xml

```diff
@@ -25,22 +25,85 @@
     </para>
    </glossdef>
   </glossentry>
+  <glossentry><glossterm>&alertmanager;</glossterm>
+   <glossdef>
+    <para>
+     A single binary which handles alerts sent by the &prometheus; server and
+     notifies the end user.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>archive sync module</glossterm>
+   <glossdef>
+    <para>
+     Module that enables creating an &ogw; zone for keeping the history of S3
+     object versions.
+    </para>
+   </glossdef>
+  </glossentry>
   <glossentry><glossterm>Bucket</glossterm>
    <glossdef>
     <para>
      A point that aggregates other nodes into a hierarchy of physical
      locations.
     </para>
     <important>
-     <title>Do Not Mix with S3 Buckets</title>
      <para>
-      S3 <emphasis>buckets</emphasis> or <emphasis>containers</emphasis>
-      represent different terms meaning <emphasis>folders</emphasis> for
-      storing objects.
+      Not to be confused with S3 <emphasis>buckets</emphasis> or
+      <emphasis>containers</emphasis>.
      </para>
     </important>
    </glossdef>
   </glossentry>
+  <glossentry><glossterm>&cephadm;</glossterm>
+   <glossdef>
+    <para>
+     &cephadm; deploys and manages a &ceph; cluster by connecting to hosts from
+     the manager daemon via SSH to add, remove, or update &ceph; daemon
+     containers.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&cephfs;</glossterm>
+   <glossdef>
+    <para>
+     The &ceph; filesystem.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>CephX</glossterm>
+   <glossdef>
+    <para>
+     The &ceph; authentication protocol. Cephx operates like Kerberos, but it
+     has no single point of failure.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&ceph; Client</glossterm>
+   <glossdef>
+    <para>
+     The collection of &ceph; components which can access a &ceph; Storage
+     Cluster. These include the &ogw;, the Ceph Block Device, the &cephfs;, and
+     their corresponding libraries, kernel modules, and FUSEs.
```

Comment:
What are FUSEs here?

Reply:
https://docs.ceph.com/en/latest/man/8/ceph-fuse/

Reply:
OK. I suspected as much. So why it is plural?

How  about "FUSE clients" instead?